### ISSUE TITLE
Progressbar: Accessibility attribute missing

### ISSUE THREAD

**Issue by daniel-cpl** [primefaces/primeng#11460]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Plunkr Case (Bug Reports)**
Please demonstrate your case at stackblitz by using the issue template below. Issues without a test case have much less possibility to be reviewd in detail and assisted.
**Current behavior**
The progressbar component misses an aria-attribute to pass the accessibility test criteria. See [https://dequeuniversity.com/rules/axe/4.4/aria-progressbar-name?application=axeAPI](https://dequeuniversity.com/rules/axe/4.4/aria-progressbar-name?application=axeAPI) for reference.

**Expected behavior**
The progressbar component should have an aria-attribute to pass the accessibility test criteria.

**Minimal reproduction of the problem with instructions**
Inspect HTML from one of the progressbar examples [https://www.primefaces.org/primeng/progressbar](https://www.primefaces.org/primeng/progressbar)

Element `<div role="progressbar">` should have the aria-attribute.

**What is the motivation / use case for changing the behavior?**
The progressbar should work properly with screen readers and progressbar accessibility tests should pass.

**Please tell us about your environment:**
MacOS, VS code

* **Angular version:** 13.2.6

* **PrimeNG version:** 13.3.0

* **Browser:** [all]

 
* **Language:** [TypeScript 4.5.5]

**Comment by mertsincan** [comment]
Hi,

So sorry for the delayed response! Improvements have been made to many components recently, both in terms of performance and enhancement. Therefore, this improvement may have been developed in another issue ticket without realizing it. You can check this in the documentation. If there is no improvement on this, can you reopen the issue so we can include it in our roadmap? 
Please don't forget to add your feedback as a comment after reopening the issue. These will be taken into account by us and will contribute to the development of this feature. Thanks a lot for your understanding!

Best Regards,
